src/test/java/gov/usgs/wma/waterdata/collections/ObservationsStatTimeSeriesControllerIT.java

```diff
@@ -71,7 +71,7 @@ public void timeSeriesNotInFeatureTest() throws Exception {
 	@Test
 	public void notFoundTimeSeriesTest() {
 		String url = makeURL(DEFAULT_COLLECTION_ID, "USGS-12345678", "216d009de8914147a0f9e5237da77854");
-//		ResponseEntity<String> response = restTemplate.getForEntity("/collections/monitoring-locations/items/USGS-12345678/time-series/216d009de8914147a0f9e5237da77854", String.class);
+//		String url = "/collections/monitoring-locations/items/USGS-12345678/time-series/216d009de8914147a0f9e5237da77854", String.class);
```

Comment:
What is the purpose of a function to build a url when the actual expected url is embedded in a comment?

Reply:
That comment should be deleted. I will check. The reason that URL is commented out is because it is wrong but contained the params I needed. I will check, but I thought I removed them all in a later commit. might or must have missed some.

Reply:
I thought I had already comment, but do not see them. Would agree, in general is cleaner to not have commented out code.

Reply:
removed